tools/enzyme/README.md

```diff
@@ -5,3 +5,23 @@
 Enzyme needs to be compiled against a specific version of LLVM. You are strongly advised to use the Dockerfile for this one.
 
 [Enzyme]: https://enzyme.mit.edu/
+
+## Commentary
+
+### Multithreading
+
+The approach here is very similar to the one for [manual](../manual),
+except that the derivative functions are (obviously) produced with
+Enzyme. In particular, the primal functions use the [C++ reference
+implementations][../../cpp/gradbench/evals] - see the Commentary for
```

Comment:
This link is broken; please fix it and also replace the relative link with one using a leading slash.